### PR TITLE
Kahu build system

### LINKED ISSUE #53: Support Kahu build system
**Issue/Feature Description:**
Feature handle kahu build system
- [x] Basic static code checker (go fmt, go vet, goimports)
- [x] Build binaries
- [x] Build images

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
Improve kahu build experience 
**How to reproduce, in case of a bug:**
NA
**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**
NA

### PR DESCRIPTION
**Pull request by AmitRoushan**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

/kind enhancement

**What this PR does / why we need it**:

The PR adds scripts to verify, build and release process of kahu binary and images.

**Which issue(s) this PR fixes**:

Fixes #53

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:
Build command options:
image

image

image

image

**Special notes for your reviewer**:

### CODE REVIEW

File: Makefile.generate_proto [hunk 1]
```diff
@@ -0,0 +1 @@
+build/Makefile.generate_proto
```
Comment (sushanthakumar) [inline Makefile.generate_proto:1]: Pls add one Readme file with all the support provided by this make file

Comment (AmitRoushan) [reply Makefile.generate_proto:1]: The issue is tracked in #54

**Review by joseph-v — APPROVED** [review]
LGTM

**Comment by sushanthakumar** [comment]
Pls add build option examples in PR test section for usage

**Review by sushanthakumar — APPROVED** [review]

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

**Pull request merged**